packages/cli/src/commands/deploy.ts

```diff
@@ -116,6 +116,18 @@ export default class Deploy extends AuthCommand {
       if (!preview) {
         await ux.wait(500)
         this.log(`Successfully deployed project "${project.name}" to account "${account.name}".`)
+
+        // Print the ping URL for heartbeat checks.
+        const heartbeatLogicalIds = project.getHeartbeatLogicalIds()
+        const heartbeatCheckIds = data.diff.filter((check) => heartbeatLogicalIds.includes(check.logicalId))
+          .map(check => check?.physicalId)
+
+        heartbeatCheckIds.forEach(async (id) => {
+          if (id && typeof id === 'string') {
```

Comment:
Which case is this btw?

Reply:
This is when the `physicalId` is the `checkId` (in this case of a heartbeat check), this is always going to be a string since it's a uuid

Reply:
Do we need this if then?

Reply:
Typescript was otherwise complaining on the usage of the id right below this line

Reply:
Ah, gotcha. You can cast it to string using `as` btw. It will be always string in heartbeats so we can force it here